src/_base/harness/config/commands.yml

```
@@ -202,3 +202,9 @@ command('feature xdebug cli (on|off)'):
     run ws service php-fpm restart
     echo 'Done'
 
+command('db import <database_file>'):
+  env:
+    COMPOSE_PROJECT_NAME: = @('namespace')
+  exec: |
+    #!bash(workspace:/)|@
```

To use `={input.argument('database_file')}`, this needs to be:
```suggestion
    #!bash(workspace:/)|=
```

---

The thing is.... it does work?

---

So something wrong with the ws code expected result.  I will still change it as I can see the intention.